Makefile

```diff
@@ -32,30 +32,30 @@ setup_env:
 	@# The keyring install is to allow pip to fetch credentials for our internal repo if needed
 	PIP_INDEX_URL=https://pypi.org/simple \
 	PIP_EXTRA_INDEX_URL=https://pypi.org/simple \
-	poetry run python --version
-	poetry run python -m pip install keyring
-	poetry run python -m pip install -U pip wheel
+	python -m poetry run python --version
```

Comment:
hmm isnt the poetry run a safe execution to make aure we execute in the right project's environment?

Reply:
idk it broke on my systen but I'll revert this in the end yes

Reply:
reverted.